ui/apps/platform/src/Containers/NetworkGraph/NetworkGraphPage.tsx

```diff
@@ -150,97 +145,105 @@ function NetworkGraphPage() {
+                    // transform policy data to DataModel
+                    const { policyDataModel, policyNodeMap } = transformPolicyData(policyNodes);
+                    // get active nodes from network flow graph API response
+                    const { nodes: activeNodes } = values[0].response;
+                    // transform active data to DataModel
+                    const { activeDataModel, activeEdgeMap, activeNodeMap } = transformActiveData(
+                        activeNodes,
+                        policyNodeMap,
+                        namespacesFromUrl
+                    );
 
-                        // create extraneous flows graph
-                        const extraneousFlowsDataModel = createExtraneousFlowsModel(
-                            policyDataModel,
-                            activeNodeMap,
-                            activeEdgeMap,
-                            namespacesFromUrl
-                        );
+                    // create extraneous flows graph
+                    const extraneousFlowsDataModel = createExtraneousFlowsModel(
+                        policyDataModel,
+                        activeNodeMap,
+                        activeEdgeMap,
+                        namespacesFromUrl
+                    );
 
-                        const newUpdatedTimestamp = new Date();
-                        // show only hours and minutes, use options with the default locale - use an empty array
-                        const lastUpdatedDisplayTime = newUpdatedTimestamp.toLocaleTimeString([], {
-                            hour: 'numeric',
-                            minute: '2-digit',
-                        });
-                        setLastUpdatedTime(lastUpdatedDisplayTime);
-                        setPrevEpochCount(epoch);
-                        setCurrentEpochCount(epoch);
+                    const newUpdatedTimestamp = new Date();
+                    // show only hours and minutes, use options with the default locale - use an empty array
+                    const lastUpdatedDisplayTime = newUpdatedTimestamp.toLocaleTimeString([], {
+                        hour: 'numeric',
+                        minute: '2-digit',
+                    });
+                    setLastUpdatedTime(lastUpdatedDisplayTime);
 
-                        setModels({
-                            activeModel: activeDataModel,
-                            extraneousModel: extraneousFlowsDataModel,
-                        });
-                    })
-                    .catch(() => {
-                        // TODO
-                    })
-                    .finally(() => setIsLoading(false));
-            }
+                    // Set the epoch to the most recent value from the server, since the state should now be up to date
+                    // with that value at worst.
+                    setPrevEpochCount(currentEpochCount);
```

Comment:
At best, this correctly tracks that the graph is up to date.

At worst, there was a change in between the time the request was sent and the time this value was updated, which will show the "# updates available" button to the user on the next poll.